_posts/2022-03-23-#21778.md

```diff
@@ -10,14 +10,39 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+Historically Bitcoin Core has used a very ["non-standard" toolchain](https://github.com/bitcoin/bitcoin/blob/master/contrib/macdeploy/README.md) for producing its macOS release binaries. This is primarily due to there not being any "official" way to build macOS binaries, using completely open source software, on a Linux system. The only Apple-sanctioned way to build macOS binaries is using the Xcode toolchain, running on macOS hardware.
+
+Using a non-standard toolchain is not ideal for a number of reasons:
+    * We are the only user. No one else is testing things / finding bugs / making improvements.
+    * Creating our toolchain is reliant on Apple (infrequently) [releasing source code](https://opensource.apple.com/source/) for the tools we need.
+    * It's reliant on 3rd-parties patching those sources. i.e https://github.com/tpoechtrager/cctools-port
+    * It's complicated.
```

Comment:
Needs newlines in between, otherwise they're turned into asterisks instead of bullet points